README.md

```diff
@@ -15,6 +15,7 @@ and then loaded by lk2nd.
 ## Supported SoCs
 - MSM8916
 - MSM8929
+- MSM8216
 - MSM8939
```

Comment:
I think we should keep this list alphabetically ordered.

Reply:
Done